### DISCUSSION TITLE
🐙 End-to-End Airbyte Source Connector Migration from Python CDK to Low Code

### DISCUSSION THREAD

**Discussion by topefolorunso** [airbytehq/airbyte#29995] [Show and tell]
Hey community folks! 👋

Following the Airbyte August connector hackathon, I am excited to share a simple step-by-step guide to migrating an Airbyte source connector from Python CDK to a Low Code CDK.

Find the medium article [here - medium.com/@topefolorunso/end-to-end-migration](https://medium.com/@topefolorunso/migrating-an-airbyte-source-connector-from-python-cdk-to-the-low-code-cdk-the-confluence-source-f9213d3d989e)

The article also contains instructions for testing and publishing the migrated connector.

Kindly share your thoughts and feedback as you enjoy it. I indicated possible issues that Windows users may encounter. However, If you encounter other issues, feel free to comment under this discussion and we can take it up from there.

Cheers! ✨

-- Tope

_P.S: Please give this GitHub discussion an upvote and help me win the tutorials contest. Thank you!_

**Comment by possibiliterian** [comment]
Nice 👍

**Reply by Oladoyin122** [reply]
Awesome 👍👍

**Reply by Obacruz** [reply]
Nice

**Reply by Layomibaba20** [reply]
Love it!

**Reply by Lilian-oc** [reply]
Niiiice

**Reply by XX-001** [reply]
Amazing!

**Reply by GboyegaAkint** [reply]
Nice, the steps are easy to follow.

**Reply by Itunuoluwatomiwa** [reply]
Just what I have been looking for, thank you❤️

**Reply by Miwaa4** [reply]
Thank you for sharing❤️

**Reply by Dorcasola4** [reply]
Great article, keep it up❤️

**Reply by FavourOTL** [reply]
Nice bro

**Comment by joshuaitakpe** [comment]
Good one!

**Reply by durupraise** [reply]
Awesome 👍😊

**Comment by willnotsin** [comment]
Lovely stuff

**Comment by DairoIyin** [comment]
Good job.

**Comment by teema17** [comment]
Well done!!!

**Comment by Bidi1234** [comment]
This is really nice

**Comment by Topesbabe** [comment]
Superb!

**Reply by Topesbabe** [reply]
Glad I stumbled on this

**Comment by ma-yowa** [comment]
Nice 😊

**Comment by kerdaino** [comment]
Wonderful ☺️

**Comment by ChocoBabyy** [comment]
Loveeee thisss!🔥

**Reply by Olufems** [reply]
Nice break down, well done chief.

**Comment by Olusoga** [comment]
Good one
